### ISSUE TITLE
Demo Version Not working

### ISSUE THREAD

**Issue by ManiMatter** [1014156094/vue-audio-player#71]
When I click on the [Online Demo](https://codesandbox.io/s/liripengvue-audio-player-issue-moban-cb57s?file=/src/App.vue&resolutionWidth=320&resolutionHeight=675) link in the ReadMe, the CodeSandbox tries to open but never loads fully. 

Here's what I see with the console open (on Chrome 125.0.6422.114 on iOS 14.4 Sonoma):

<img width="700" alt="image" src="https://github.com/1014156094/vue-audio-player/assets/124743318/8cf7c04c-ff8b-49c8-ad92-6910a10af47e">

**Comment by 1014156094** [comment]
Try this link https://github.com/1014156094/vue-audio-player/blob/master/pay.jpg?raw=true

**Comment by ManiMatter** [comment]
Thank you. The link works for me from my Mac Laptop.
However, from my iPhone I see a somewhat weird behavior:

First, the codebox installs:
![IMG_7664](https://github.com/1014156094/vue-audio-player/assets/124743318/0396981a-f4dd-41f5-a7ea-a7e4b89d933d)

Then I see the player (not playing):
![IMG_7665](https://github.com/1014156094/vue-audio-player/assets/124743318/edfafdfb-3b9a-412e-b1ab-8c13f50337df)

When I click on the play button, I see this error:
![IMG_7666](https://github.com/1014156094/vue-audio-player/assets/124743318/70613370-4ec9-4295-b034-f96d7ab821a7)

When I close the error and click play again, it starts playing:


![IMG_7667](https://github.com/1014156094/vue-audio-player/assets/124743318/0cd648cd-4bfa-42ec-bcde-8936d00d4a1c)

However, when I lock my iphone screen, the play gets immediately stopped. 


In summary, I see two problems
1) The error message when clicking play the first time
2) That the player stops when the iPhone locks

Is that expected behavior in the CodeSandbox and normally it should work?

**Comment by 1014156094** [comment]
Defects of CodeSandbox，I have no problem trying in the vue program
